### PR TITLE
Remove hardcoded pronoun

### PR DESCRIPTION
**Pull request by mattstratton**
In the current implementation, assumes the user is a "her". This adds a configuration item to set the pronoun, i.e. "her", "his", or "their".

### CODE REVIEW

**Comment by robinske** [comment]
Good call! Can you squash commits? It's giving a weird state (showing some changes that've already been merged)

File: wthik.py [hunk 1]
```diff
@@ -127,17 +124,22 @@ def travel_schedule(service, calendar_id, num_events=5):
 
 def help_response():
     resp = MessagingResponse()
-    resp.message("""Ask me "Where is Kelley?" to see my current whereabouts or "Travel schedule" to see what's coming up.""")
+    resp.message("""Ask me "Where is Matt?" to see my current whereabouts or "Travel schedule" to see what's coming up.""")
```
Comment (robinske) [inline wthik.py:127]: Let's update this to `TRAVELER` while we're at it.

File: wthik.py [hunk 2]
```diff
@@ -127,17 +124,22 @@ def travel_schedule(service, calendar_id, num_events=5):
...
     return str(resp)
 
+@app.route("/test")
+def test():
+    service = _build_service()
+    calendar_id = app.config.get("CALENDAR_ID")
+    return where_is_she(service, calendar_id)
+
```
Comment (robinske) [inline wthik.py:135]: this is removed in master, squashing might get rid of this (was in the branch I had previously shared with you)

File: wthik.py (hunk 1 above)
Comment (mattstratton) [reply wthik.py:127]: Oh yeah, I missed that one.

**Comment by mattstratton** [comment]
I'm thinking of whacking this PR and starting a new one (after a rebase on my end) since I think that a lot of the stuff that is in app_config should probably be moved to env vars anyway (like the name, pronoun, etc). What do you think?

**Comment by robinske** [comment]
I'm ok to leave name + pronoun in the config file since it's not privileged info (others may feel differently?) & there's already a lot of env variables to manage.

**Pull request closed without merging**